### PR TITLE
page-header murcked up&stylized

### PR DESCRIPTION
**Pull request by Olena-Horobets**
Guys, check out my awesome header! Also for some reason logo doesn't seem to be align vertically. Have I done something wrong? If so, let me know, please))

### CODE REVIEW

File: src/sass/_common.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.container {
+  width: 1200px;
```
Comment (vladibon) [inline src/sass/_common.scss:2]: For better appearance, you should use max-width

File: src/sass/_page-header.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+.page-header {
+  background-color: $bgr-dark-color;
+  height: 70px;
```
Comment (vladibon) [inline src/sass/_page-header.scss:3]: Don’t use height in this case.

File: src/sass/_page-header.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  transition: color 250ms ease-out, transform 250ms ease-out;
+
+  &:hover,
+  &:focus {
+    color: $accent-color;
+    transform: scale(1.1);
+  }
+}
+
+.main-nav__list {
```
Comment (vladibon) [inline src/sass/_page-header.scss:30]: Add also padding-left: 0; for main-nav__list

**Review by vladibon — CHANGES_REQUESTED** [review]
Your awesome header is perfect! It needs just a few fixes.

File: src/sass/_page-header.scss (hunk 3 above)
Comment (Olena-Horobets) [reply src/sass/_page-header.scss:31]: thank you, I forgot)

**Review by vladibon — APPROVED** [review]
That's really awesome now!

**Review by Valeriy-Fedulov — APPROVED** [review]

**Pull request merged**